src/app/pages/buyer/buyer-bid-list/buyer-bid-list.component.ts

```diff
@@ -47,7 +47,7 @@ export class BuyerBidListComponent implements OnInit, OnDestroy {
    */
   private initializeBidList(buyerId: number) {
     this.appServ.getBidsForBuyer(buyerId).subscribe(data => {
-      this.bids = data;
+      this.bids = data.sort((a, b) => b.id - a.id)
```

Comment:
@subbu3012 - sorting of data with paginated results won't work. Shouldn't this sort be applied at the backend?

Reply:
@vraravam to be frank, both sorting and pagination should be done on backend. doing this on frontend has two reasons:
1. we havent done pagination on backend (yet). 
2. azhagu wanted this to be done very badly as there is a drastic incoming of bids after launch. 

coming back to how this code works, the sorting is done on the complete data (the apis response) and then, that sorted data is paginated. the reverse (paginating and then sorting) wont work as you said.

Reply:
Yes-  now that I see the code - the backend doesn't do both these functions. We should play cards to accomplish both these CFRs in the backend. Approving with that being considered as "tech debt"